apps/vr-tests/src/stories/CompoundButtonNext.stories.tsx

```diff
@@ -0,0 +1,535 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import Screener, { Steps } from 'screener-storybook/src/screener';
+import { storiesOf } from '@storybook/react';
+import { CompoundButton } from '@fluentui/react-button';
+import { AddIcon } from '@fluentui/react-icons';
+import { withThemeProvider } from '@fluentui/storybook';
+import { FabricDecorator, FabricDecoratorFullWidth } from '../utilities';
+
+storiesOf('CompoundButton Next', module)
```

Comment:
could you "copy" splitButtonNext tests for these new tests? I simplified the tests for variations to avoid adding too many tests which are not very value-able and cost us build test

Reply:
Done.